packages/react-components/src/components/UnsupportedEnvironment.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { Icon, Link, Stack, Text } from '@fluentui/react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import {
+  containerStyles,
+  iconStyles,
+  linkTextStyles,
+  mainTextStyles,
+  secondaryTextStyles
+} from './styles/UnsupportedEnvironment.styles';
+import { UnsupportedBrowserStrings } from './UnsupportedBrowser';
+import { UnsupportedBrowserVersionStrings } from './UnsupportedBrowserVersion';
+
+/**
+ * props for UnsupportedBrowser UI
+ *
+ * @beta
```

Comment:
This should be `@private`.

A component in the API that accepts `UnsupportedBrowserStrings | UnsupportedBrowserVersionStrings` doesn't make sense to me. This reads more like a `@private` helper component.